src/django/api/views/auth.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import status
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+from rest_framework.exceptions import AuthenticationFailed
+from dj_rest_auth.views import LoginView, LogoutView
+
+from django.contrib.auth import authenticate, login, logout
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.utils.decorators import method_decorator
+
+
+class Login(LoginView):
+    permission_classes = (AllowAny,)
+
+    @method_decorator(ensure_csrf_cookie)
```

Comment:
Is this required here? Since we already have the middleware installed?

https://github.com/azavea/iow-boundary-tool/blob/8ee6d5423fad7e6e577bfc80a8df4cd829ba8b8a/src/django/iow/settings.py#L108

Reply:
It was required with my first attempt at axios integration, but I'll give it another look if still needed.

Reply:
From the [CSRF reference](https://docs.djangoproject.com/en/dev/ref/csrf/), it's not clear to me that the cookie is always sent with the response. Before posting the PR, I found that re-logging in at /login would sometimes fail, as the user was already in an authenticated session, and adding `ensure_csrf_cookie` fixed it. Later, using the correct `axios` integration seemed to take care of that, and plus--now you are redirected away from `/login` if you're logged in. So I'll take it out 👍🏻